foxmlToSolr.xslt

```diff
@@ -113,7 +113,33 @@
   <!-- Used to index the list of collections to which an object belongs.
     Requires the "traverse-graph.xslt" bit as well. -->
   <xsl:include href="/usr/local/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/hierarchy.xslt"/>
+  <!-- begin: islandora_solution_pack_oralhistories setup -->
+  <xsl:include href="/usr/local/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/or_transcript_solr.xslt"/>
+  <xsl:include href="/usr/local/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/vtt_solr.xslt"/>
+  <!-- end: islandora_solution_pack_oralhistories setup --> 
+  
+  	<xsl:template match="/">
```

Comment:
... I don't think this actually works. We have two `<xsl:template match="/">` statements both implementing `<xsl:apply-templates select="foxml:datastreamVersion[last()]">` on a slight variant of an `<xsl:when>` statement. One or the other is gonna get squashed. This appears to be directly copy-pasted from https://github.com/digitalutsc/islandora_solution_pack_oralhistories/wiki/Configuration:--Basic-Indexing-of-transcripts-in-Solr ... but I'm not sure that the instructions there are quite valid.